feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fefebad99209211ec30537.md

---
id: 64fefebad99209211ec30537
title: Step 49
challengeType: 0
dashedName: step-49
---

# --description--

> If you check the `Application` tab of your browser console, you'll see a series of `[object Object]`. That's because everything you save in `localStorage` has to be a string.

Reviewer: ```suggestion
If you check the `Application` tab of your browser console, you'll notice a series of `[object Object]`. This is because everything you save in `localStorage` needs to be in string format.
```

> To fix the issue, wrap the data you're saving in the `JSON.stringify()` method, then check local storage again to see what happens.

Reviewer: ```suggestion
To resolve the issue, wrap the data you're saving in the `JSON.stringify()` method. Then, check local storage again to observe the results.
```
